projects/rero/ng-core/src/lib/record/editor/type/custom-select/custom-select.component.ts

```diff
@@ -140,6 +140,19 @@ export class CustomSelectFieldComponent extends FieldType implements OnDestroy,
    * @returns The selected values as string.
    */
   get selectedValuesAsString(): string {
+    if (Array.isArray(this.formControl.value)) {
+      // Keeps selected items in order
+      const data = [];
+      let optionSelected: SelectOption;
+      const selectedValue = this.formControl.value.filter((v: string) => v !== undefined);
+      selectedValue.forEach((val: string) => {
+        optionSelected = this.selectedOptions
```

Comment:
I think in correct english it should be `selectedOption` (as `selectedValue` just line above)

Reply:
My English is not good and it shows in the code 🙈😂